java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -667,4 +668,41 @@ CompletableFuture<Map<String, Double>> zrangeWithScores(
      * }</pre>
      */
     CompletableFuture<Long> zlexcount(String key, LexRange minLex, LexRange maxLex);
+
+    /**
+     * Returns the number of members in the intersection of sorted sets specified by <code>keys</code>
+     * .
+     *
+     * @see <a href="https://redis.io/commands/zintercard/">redis.io</a> for more details.
+     * @param keys The keys of sorted sets to intersect.
+     * @param limit Specifies a maximum number for the intersection cardinality; if this number is
```

Comment:
remove: `if this number is reached the process stops and returns the limit` as too much detail.